src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -2,7 +2,7 @@
   <div
     ref="builder-toolbar"
     class="tree-builder-toolbar d-flex flex-column">
-    <div class="tree-builder-toolbar-main-menu d-flex flex-column">
+    <div class="tree-builder-toolbar tree-builder-toolbar-main-menu d-flex flex-column">
```

Comment:
This can be undone. There is already a root element with the class we need.

Reply:
:heavy_check_mark: